lib/rubocop/cop/sorbet/one_ancestor_per_line.rb

```diff
@@ -0,0 +1,81 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'rubocop'
+
+module RuboCop
+  module Cop
+    module Sorbet
+      # This cop ensures one ancestor per requires_ancestor line
+      # rather than chaining them as a comma-separated list.
+      #
+      # @example
+      #
+      #   # bad
+      #   module SomeModule
+      #     requires_ancestor Kernel, Minitest::Assertions
+      #   end
+      #
+      #   # good
+      #   module SomeModule
+      #     requires_ancestor Kernel
+      #     requires_ancestor Minitest::Assertions
+      #   end
+      class OneAncestorPerLine < RuboCop::Cop::Cop
+        MSG = 'Cannot declare more than one ancestor per line'
+
+        def_node_search :requires_ancestors, <<~PATTERN
+          (send nil? :requires_ancestor ...)
```

Comment:
I think we only one to correct them if called on `self` (so always no receiver):

```suggestion
          (send nil :requires_ancestor ...)
```

Reply:
Makes sense! I'll fix these in the other places too.

Reply:
Hmm if I change these, all of my tests fail.
Will dig into it a bit.

Reply:
My  bad, `nil` is actually the `nil` value... 🤦